modules/tinymce/src/core/main/ts/api/Formatter.ts

```diff
@@ -132,9 +132,10 @@ const Formatter = (editor: Editor): Formatter => {
      * @param {String} name Name of format to match.
      * @param {Object} vars Optional list of variables to replace before checking it.
      * @param {Node} node Optional node to check.
+     * @param {Boolean} similar True/false state if the match should handle similar or exact formats.
```

Comment:
```suggestion
     * @param {Boolean} similar Optional argument to specify that similar formats should be checked instead of only exact formats.
```